.eslintrc.js

```diff
@@ -1,31 +1,28 @@
 module.exports = {
-  ignorePatterns: ['.eslintrc.js'],
+  ignorePatterns: ['.eslintrc.js', 'globals.d.ts', 'webpack.config.js'],
```

Comment:
Proč se tady vyhazuje to `globals.d.ts`? To klidně můžete eslintem nechat kontrolovat. Webpack config tu máte dobře.